docs/product-assembly_pam-on-openshift/src/main/asciidoc/environment-authoring-single-proc.adoc

```diff
@@ -0,0 +1,35 @@
+[id='environment-authoring-single-proc']
+= Deploying a single authoring environment
+
+To deploy a single authoring environment, use the `rhpam-70-authoring.yaml` template file. You can download this file from `{URL_BASE_GITHUB_PAM}/templates`.
+
+.Procedure
+
+. Use one of the following methods to deploy the template:
+.. In the OpenShift Web UI, select *Add to Project >  Import YAML / JSON* and then select or paste the `rhpam-70-authoring.yaml` file. In the *Add Template* window, ensure *Process the template* is selected and click *Continue*.
+.. To use the OpenShift command line console, prepare the following command line:
++
+[subs="verbatim,macros"]
+----
+oc process -n <project-name> -f <template-path>/hpam-70-authorimg.yaml -v ATTRIBUTE=value -V ATTRIBUTE=value... | oc create -n <project-name> -f -
+----
++
+In this command line:
++
+* Replace `<project-name>` with the name of your project
+* Replace `<template-path>` with the path to the downloaded template file.
+* Use as many `ATTRIBUTE=value` pairs as needed to set the required attributes. You can view the template file to see descriptions for the attributes.
++
+. Set the following mandatory attributes:
```

Comment:
This is really a problem on our side, but the whole concept of a "mandatory" template parameter with a default value is somewhat meaningless.